templates/connect-inject-deployment.yaml

```
@@ -41,6 +41,35 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            {{- /* A Consul client and ACL token is only necessary for the connect injector if namespaces are enabled */}}
+            {{- if .Values.global.enableConsulNamespaces }}
+            - name: HOST_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
+            {{- if (and .Values.connectInject.aclInjectToken.secretName .Values.connectInject.aclInjectToken.secretKey) }}
+            - name: CONSUL_HTTP_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.connectInject.aclInjectToken.secretName }}
+                  key: {{ .Values.connectInject.aclInjectToken.secretKey }}
+            {{- else if .Values.global.bootstrapACLs }}
+            - name: CONSUL_HTTP_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ template "consul.fullname" . }}-connect-injector-acl-token"
```

```suggestion
                  name: "{{ template "consul.fullname" . }}-connect-inject-acl-token"
```

This is the wrong name.

---

Updated.